charts/mageai/templates/deployment-scheduler.yaml

```diff
@@ -120,7 +120,7 @@ spec:
       nodeSelector:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      {{- with .Values.affinity }}
+      {{- with .Values.scheduler.affinity }}
```

Comment:
can you make it backward compatible?

Reply:
@wangxiaoyou1993 Could you please give me more details for backward compatible?

Reply:
the old values.yaml uses affinity config. can you make it as the fallback config here.
similar change in another PR: https://github.com/mage-ai/helm-charts/pull/45/files

Reply:
@wangxiaoyou1993 I added the backward compatibility for `affinity` value using "if else" condition.
Could you please review this changes?

- **Related commit**: 04084d13d3316a4e04013b8b79dfea604953e4e3, fa71a346d77c06ffdcde0eafa5bd1309a18f6530